helpers/dateTime.js

```diff
@@ -1,43 +1,125 @@
 import { convertUTCToLocalDateTime, getDateTimeDescriptor } from '@brightspace-ui/intl/lib/dateTime.js';
 
+// val is an object containing year, month, date
```

Comment:
I updated this file to have more consistency with naming and what functions return. `parse*` functions now all return objects, as does `getToday`. 2 functions return `Date`s and that is specified in their names.